.gitignore

```
@@ -180,3 +180,9 @@ model-cache/
 openretina_cache_folder/
 openretina_assets/
 test_data/
+mlflow/
+multirun/
+notebooks/lightning_logs
+notebooks/notebook_example/tensorboard
+checkpoints/
+tensorboard/
```

```suggestion
tensorboard/

```

Nitpick: It's the Posix standard to end files in a new line. (if you are interested why, see here for an explanation: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline :) )